src/concerto/scalar.cto

```diff
@@ -0,0 +1,3 @@
+namespace concerto.scalar@1.0.0
+
```

Comment:
Please add `concerto version ">= 3.2.0"` pragma to this file — scalars were added in Concerto version 3.2.0.